src/parser.ts

```diff
@@ -207,6 +210,25 @@ export class Parser {
         this.jobs.forEach((job) => {
             job.producers = Producers.init(this.jobs, this.stages, job);
         });
+
+        if (this.argv.enableJsonSchemaValidation) {
```

Comment:
added a flag to disable json schema validation just in case, there's some bug, 

but for now it seemed to work fine with all test cases

Reply:
I suggest having this flag enabled by default for a period of lets say 1-2 month, and then we can enable it via `GCL_ENABLE_JSON_SCHEMA_VALIDATION` in our `~/.bashrc` on our local environments to mature this feature.

Reply:
> I suggest having this flag enabled by default for a period

do you mean to disable the json schema validation by default ?

Reply:
Yes...